docs/tutorial/tutorial-7.rst

```diff
@@ -342,7 +347,8 @@ the settings for your app::
     [tool.briefcase.app.helloworld]
     formal_name = "Hello World"
     description = "A Tutorial app"
-    icon = "src/helloworld/resources/helloworld"
```

Comment:
Removal of `icon` depends on the outcome of the discussion in https://github.com/beeware/briefcase-template/pull/111.